table/metadata.go

```diff
@@ -1145,6 +1146,9 @@ func (m *metadataV1) UnmarshalJSON(b []byte) error {
 	type Alias metadataV1
 	aux := (*Alias)(m)
 
+	// Set LastColumnId to -1 to indicate that it is not set as LastColumnId = 0 is a valid value for when no schema is present
+	aux.LastColumnId = -1
```

Comment:
`last-column-id` is a required attribute, so it should never be missing. So I don't think we need to do this